scitran/data/medimg/pfile.py

```diff
@@ -413,8 +406,20 @@ def _full_parse(self, filepath=None):
 
         log.debug('_full_parse of %s' % filepath)
         try:
-            pfile = getattr(__import__('pfile.pfile%d' % self.version, globals()), 'pfile%d' % self.version)
+            print os.path.abspath(os.curdir)
+            pfile = getattr(__import__('gepfile.pfile%d' % self.version, globals()), 'pfile%d' % self.version)
         except ImportError:
+            try:
+                import gepfile
+            except ImportError:
+                raise ImportError("""
+                gepfile module not found.
+                gepfile should be installed cloning the PRIVATE repo pfile in the folder scitran/data/medimg/gepfile
+
+                from scitran.data root folder exec the following:
+
+                git clone https://github.com/cni/pfile.git && mv pfile scitran/data/medimg/gepfile
+                """)
```

Comment:
I don't think we need nested exception handling here. Having the original import fail is reason enough to print this message, starting with the detail about which exact version was not found.
```
No pfile parser for vX.
Pfile data loading requires the proprietary gepfile module to be installed in the scitran.data root folder.
git clone https://github.com/cni/pfile.git gepfile
```